### PR TITLE
Feat/156980247/build globalclockservice

### PR DESCRIPTION
**Pull request by stovemeerkat**

### CODE REVIEW

File: front/src/app/core/services/globalclock/globalclockservice.service.ts [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
 
   private tick(): void {
-    // TODO
+    this.fixtureService.tick();
+    this.artnetService.sendArtnet();
   }
 
   public start(): void {
-    this.timer = setInterval(this.tick, 1);
+    this.timer = setInterval(this.tick, 200);  // TODO: delay is currently quite arbitrary
```
Comment (stovemeerkat) [inline front/src/app/core/services/globalclock/globalclockservice.service.ts]: Delay value probably needs to be changed...

File: front/src/app/core/services/globalclock/globalclock.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
```
Comment (rudxde) [inline front/src/app/core/services/globalclock/globalclock.service.spec.ts]: i think we don't need a empty test yet.

File: front/src/app/core/services/globalclock/globalclock.service.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    this.artnetService = artnetService;
+  }
+
+  private tick(): void {
+    this.fixtureService.tick();
+    this.artnetService.sendArtnet();
+  }
+
+  public start(): void {
+    this.timer = setInterval(this.tick, 200);  // TODO: delay is currently quite arbitrary
```
Comment (rudxde) [inline front/src/app/core/services/globalclock/globalclock.service.ts]: i think a value of 16ms should do it.

File: front/src/app/core/services/globalclock/globalclock.service.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import { ArtnetService } from '../artnet/artnet.service';
+
+@Injectable()
+export class GlobalclockserviceService {
+  private timer: NodeJS.Timer;
+  private fixtureService: FixtureService;
+  private artnetService: ArtnetService;
+
+
+  constructor(fixtureService: FixtureService, artnetService: ArtnetService) {
```
Comment (rudxde) [inline front/src/app/core/services/globalclock/globalclock.service.ts]: you can declare the arguments as private, then you don't need to define them as extra variables!

**Review by rudxde — CHANGES_REQUESTED** [review]

File: front/src/app/core/services/globalclock/globalclock.service.spec.ts (hunk 2 above)
Comment (stovemeerkat) [reply front/src/app/core/services/globalclock/globalclock.service.spec.ts]: That's just what ng g generated...

**Comment by stovemeerkat** [comment]
Implemented all changes.

**Review by rudxde — APPROVED** [review]

**Pull request merged**